### PR TITLE
633 implement drupal home env in install documentation

### PR DESCRIPTION
**Pull request by bradfordcondon**
Issue #633 

## Type(s) of Change(s)

- [x] Updates documentation (inline or markdown files)

## Description

Documentation improvement.  We use an environmental variable for `/var/www/html` to make copy-pasting the commands easier for those installing elsewhere. 

I also made some typo correction and formatting changes as i spotted them (ensuring subsections were underlined, trying to use the code markup for paths, etc)

## Testing?
To build the site locally and look:

```
cd docs
make html
```
open `_build/index.html`: all changes are in the installation portion of the user's guide.

## Screenshots (if appropriate):

Here's an example of the notice and variable.

screen shot 2018-09-09 at 10 13 28 pm

### CODE REVIEW

File: docs/user_guide/install_tripal/manual_install/install_drupal.rst [hunk 1]
```diff
@@ -78,11 +85,11 @@ If an index.html file is present (as is the case with Ubuntu installations) you
 
 .. note::
 
-  It is extremely important the the hidden file .htaccess is also moved (note the second 'mv' command above. Check to make sure this file is there
+  It is extremely important the the hidden file ``.htaccess`` is also moved (note the second ``mv`` command above. Check to make sure this file is there:
 
-.. code-block:: bash
+  .. code-block:: bash
```
Comment (bradfordcondon) [inline docs/user_guide/install_tripal/manual_install/install_drupal.rst:86]: Note this isn't simply whitespace changing: this moves the codeblock *INSIDE* the note.  I think this is more appropriate.

**Review by laceysanderson — COMMENTED** [review]
Your changes look good 👍 

We also mention `/var/www/html` in other places though. For example, whenever we execute a tripal job:
```
drush trp-run-jobs --username=administrator --root=/var/www/html
```

We should likely change it throughout the user guide 😬

**Comment by bradfordcondon** [comment]
yeah i left it out of that command- adding the `$DRUPAL_HOME` warning to every page that runs a job seems really cumbersome.    Is there another way?

**Comment by laceysanderson** [comment]
I'm approving this PR. On further reflection -there is another way: We actually use the Tripal Daemon in the tutorial so no one has to manually run the tripal jobs command... As such, I've added that to #387 to remind me.

**Comment by spficklin** [comment]
I approve this pull request but I made some changes and I think someone else should double check it one more time.   I thought the "note" text was a bit too long for the pages on which it appeared.  So, I created a new page at the very beginning of the guide that describes the DRUPAL_HOME environment variable.  Then everywhere the variable is used on a page I altered the "note" to just be a very short reminder with a link back to that new page.  

I also agree with @laceysanderson that we should change `/var/www/html` everhwhere it is used to be `$DRUPAL_HOME` so I made that change and added the note to the top of every page that uses it.

**Comment by spficklin** [comment]
Here's what the new DRUPAL_HOME Variable page looks like:
screenshot from 2018-09-21 23-29-32


Here's a page with the note at the top:
screenshot from 2018-09-21 23-22-08

**Comment by laceysanderson** [comment]
I re-tested after @spficklin's changes and think it's 👍 My review stands. Are you content with this change @bradfordcondon? Specifically, the $DRUPAL_HOME page added at the beginning.

**Comment by bradfordcondon** [comment]
Can we maybe move the DRUPAL_HOME variable page one layer in, maybe into install_tripal?  It looks weird top-level, and it shouldnt be a problem that its hidden because it'll still be one of the first pages when clicking through and all guides that utlize the variable will link to it.

**Comment by spficklin** [comment]
I thought about putting it in the install portion but we use that variable all throughout the User's Guide not just in the installation.  What if we renamed the page to something like "Introduction" and add a bit of an explanation for what the User's Guide is and the DRUPAL_HOME setup is part of that page?

**Comment by bradfordcondon** [comment]
I'd be happy with putting it as a subheading in a less technically named section (ie introduction).  That said, because you are linking to the guide in a note at the top of each page, i dont think its a problem to put it in the install guide.  I'll leave that decision up to you.

I looked over the other changes you made and i like it 👍 thank you

**Comment by laceysanderson** [comment]
My inclination would be to nest it in with install. We do use it throughout but since we are linking to it, I think it’s ok for it not to be on the top level :shrugs:

**Comment by spficklin** [comment]
Okay, I've moved the page into the "Install Tripal" section and updated all the links:

screenshot from 2018-09-24 09-01-31

I think it's ready for merging if you two approve.

**Comment by bradfordcondon** [comment]
approved.  thats 3/3 so lets merge.

**Pull request merged**